lucene/facet/src/java/org/apache/lucene/facet/taxonomy/directory/DirectoryTaxonomyWriter.java

```diff
@@ -494,6 +496,7 @@ private int addCategoryDocument(FacetLabel categoryPath, int parent) throws IOEx
 
     fullPathField.setStringValue(FacetsConfig.pathToString(categoryPath.components, categoryPath.length));
     d.add(fullPathField);
+    d.add(new BinaryDocValuesField(Consts.FULL, new BytesRef(FacetsConfig.pathToString(categoryPath.components, categoryPath.length))));
```

Comment:
Could you factor out the `FacetsConfig.pathToString(...)` part in a new local variable and re-use that?  We use it in (at least?) two places here.

Reply:
Yep, this could be cleaner. Extracted out in the next commit